core/vm/errors.go

```diff
@@ -26,4 +26,5 @@ var (
 	ErrTraceLimitReached        = errors.New("the number of logs reached the specified limit")
 	ErrInsufficientBalance      = errors.New("insufficient balance for transfer")
 	ErrContractAddressCollision = errors.New("contract address collision")
+	ErrNoCompatibleInterpreter  = errors.New("could not find an interpreter to run this contract's code")
```

Comment:
`errors.New("no compatible interpreter")`

No need to make it more verbose than that.